lib/src/main/java/com/alphawallet/token/entity/ContractInfo.java

```diff
@@ -12,4 +12,12 @@
 {
     public String contractInterface = null;
     public Map<Integer, List<String>> addresses = new HashMap<>();
+    public Map<String, Module> eventModules = null;
+
+    public boolean hasContract(int chainId, String address)
```

Comment:
@JamesSmartCell could this be named for specifically?

Reply:
which? eventModules or hasContract?

Reply:
hasContract means has tokenscript for contract?